ray-curator/ray_curator/stages/deduplication/fuzzy/buckets_to_edges.py

```diff
@@ -46,7 +46,7 @@ class BucketsToEdgesStage(ProcessingStage[FileGroupTask, FileGroupTask]):
 
     def __init__(
         self,
-        output_dir: str,
+        output_path: str,
         doc_id_field: str = CURATOR_DEDUP_ID_STR,
```

Comment:
sort of nit now, but id_field or doc_id_field

Reply:
The reason this is `doc_id_field` is because the input df also contains `_bucket_id` referring to the bucket id.